modules/ROOT/pages/migrate-from-an-earlier-version-of-bonita.adoc

```diff
@@ -301,7 +294,6 @@ Install the new license:
 
 . Start the application server. Before you start Bonita Portal, clear your browser cache. If you do not clear the cache, you might see old, cached versions of Portal pages instead of the new version.
```

Comment:
Before you start Bonita Portal, clear your browser cache. => Before you use Bonita Applications, clear your browser cache.

Reply:
instead of the new version. => instead of the new Bonita Applications.

Reply:
We talked about this in a previous pr, the context of the phrase is update BEFORE version 7.7, where Bonita Portal still exists

Reply:
Anyways, this is not the reason for this pr, it's the removal of some pages, not renaming migration into update, nor removal of Portal from the project